File: BitwiseAndArithmeticModulo/BinaryToDecimal/binToDec.py
class Solution:
    def binaryToDecimal(self, b):
        # code here
        n = len(b)
        ans = 0
        base = 1

        for i in range(n - 1, -1, -1):
            ans += base * (ord(b[i]) - 48)
            base *= 2

        return ans

File: BitwiseAndArithmeticModulo/BinaryToDecimal/binToDec.cpp
#include <string>
using namespace std;

class Solution {
  public:
    int binaryToDecimal(string& b) {
        // code here
        int ans = 0;
        int base = 1;
        int n = b.length();
        for(int i=n-1;i>=0;i--){
            ans += base*(b[i]-48);
            base *= 2;
        }
        return ans;
    }
};